gems/pending/appliance_console/internal_database_configuration.rb

```diff
@@ -15,7 +15,7 @@ def self.postgres_dir
     end
 
     def self.postgresql_template
-      RAILS_ROOT.join("../system/TEMPLATE").join(postgres_dir)
+      PostgresAdmin.template_directory.join(postgres_dir)
```

Comment:
Agreed that it is too bad we have to rely upon template directory, but lets not revisit at this time.

Ok, so is the request to move `template_directory` method from `PostgresAdmin` to `InternalDatabaseConfiguration`?